examples/minikube/README.rst

```diff
@@ -1,4 +1,4 @@
 Getting Started with Cilium on minikube
 =======================================
 
-https://cilium.readthedocs.io/en/latest/gettingstarted/minikube
\ No newline at end of file
+https://docs.cilium.io/en/latest/gettingstarted/k8s-install-default/ - click on the "minikube" tab in the [Create the Cluster](https://docs.cilium.io/en/latest/gettingstarted/k8s-install-default/#create-the-cluster) section.
```

Comment:
Note the syntax for the link is broken because we expect reStructuredText syntax, not Markdown. But I think just pointing to the general page is enough anyway.